content/chef_compliance_phase.md

```diff
@@ -0,0 +1,427 @@
+
+#### Insecure
+
+Setting the attribute `default['audit'['insecure']` to `true` will skip SSL certificate verification for the `chef-automate` and `chef-server-automate` reporters. Default is `false`
+
+#### Reporters
+
+Reporters control what is done with the resulting report after the Chef InSpec run. Either a single value or a list of multiple values is supported. The default is the `json-file` reporter.
+
+##### chef-automate
+
+Sends the results to a Chef Automate instance. See the instructions on [using Chef Automate with Compliance Phase](#direct-reporting-to-chef-automate) for more details.
+
+##### chef-server-automate
+
+Sends the results to a Chef Automate proxied by a Chef Infra Server instance. See the instructions on [using Chef Infra Srver with Compliance Phase](#reporting-to-chef-automate-via-chef-infra-server) for more details.
+
+##### json-file
+
+Writes the results to disk in the location specified with the [json-file location attribute](#json-file-location).
+
+##### audit-enforcer
+
+A special reporter that instead of sending the results of the Chef InSpec run somewhere instead will fail the Chef Infra run if any control of any Chef InSpec profile fails.
+
+##### JSON File Location
+
+The location on disk that Chef InSpec's json reports are saved to when using the 'json-file' reporter. Set this attribute with `default['audit']['json-file']['location']`. Defaults to: `<chef_cache_path>/compliance_reports/compliance-<timestamp>.json`
+
+#### InSpec Backend Cache
+
+Chef InSpec provides the ability to cache the result of commands executed on the node being tested. This drastically improves Chef InSpec performance when slower-running commands are run multiple times during execution. This feature is enabled by default for Compliance Phase. If your Chef InSpec profile runs a command multiple times expecting different output, you may have to disable this feature. To do so set the `inspec_backend_cache` attribute to `false`:
+
+```ruby
+node.normal['audit']['inspec_backend_cache'] = false
+```
+
+#### Fetchers
+
+Fetchers control if Chef InSpec profiles should be fetched from Chef Automate or Chef Infra Server in addition to the default fetch locations provided by Chef Inspec. Set this attribute with `default['audit']['fetcher']`. Accepted values: 'chef-server' or 'chef-automate'. With nothing specified for this attribute the [fetchers included in Chef InSpec](https://docs.chef.io/inspec/profiles/#profile-dependencies) are used. The fetchers included with Chef Infra are mutually exclusive so only one value can be specifed for this attribute.
```

Comment:
This links to a section on profile dependencies because I could not find any more appropriate documentation on how to consume pre-existing profiles in InSpec.